build/release.sh

```diff
@@ -236,6 +253,8 @@ while getopts "mnp" opt; do
     esac
 done
 
+# Before running release, make sure using correct version of NodeJS
+validate_node_version;
```

Comment:
this might make more sense added to the push_new_release function like we do with other checks?

Reply:
Done!